scripts/install/readarr.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+# readarr installer
+# Bakerboy448 for swizzin 2020
+# Based on Radarr v3 installationn by flying sausages
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+_install_readarr() {
+    apt_install curl sqlite3
+
+    readarrConfDir="/home/$readarrOwner/.config/readarr"
+    mkdir -p "$readarrConfDir"
+    chown -R "$readarrOwner":"$readarrOwner" /home/$readarrOwner/.config
+
+    echo_progress_start "Downloading source files"
+    if ! curl "https://readarr.servarr.com/v1/update/nightly/updatefile?os=linux&runtime=netcore&arch=x64" -L -o /tmp/readarr.tar.gz >> "$log" 2>&1; then
```

Comment:
Any chance you can add a switch axe for arm64?

Reply:
can do; there code to borrow that gets the OS info?

Reply:
should be available to you as it gets sourced from the `functions/os` file

Reply:
for me, when I run `_os_arch` on my M1's VM I get `arm64` for example